src/App.svelte

```diff
@@ -36,17 +35,20 @@
 </script>
 
 <main>
-  <h3>Settings</h3>
+  <span class="block text-5xl text-center w-full py-3">Video 2 SRT</span>
+  <div class="flex flex-row gap-3 pb-3">
   <Model bind:useWhisper={useWhisper} bind:WHISPER_RETURN_DATA={whisper_captions} bind:STORED_MODEL={stored_model}></Model>
   <Languages bind:value={language}></Languages>
+  </div>
   {#if stored_model}
     <p style="color:green">Model Ready to use!</p>
   {/if}
   <hr>
   <FileHandler bind:audio_data={audio_data}></FileHandler>
   <hr>
   {#if whisper_captions == 0 && window.SUB_DATA.length == 0}
-    <CircularProgress></CircularProgress>
+  <!-- arbitary values currently you can update this with the real variables recieved from svelte store  -->
+    <div class="radial-progress" style="--value:70;">70%</div>
```

Comment:
```suggestion
  <!--  <div class="radial-progress" style="--value:70;">70%</div> --->
```
I need to find a way to make a progress bar. Let's not clutter space for now :D

Reply:
Oh that's fine, You will just need to store and update one variable that shows progress and the ui can take it from there just le know when you are done with that 👀